config/routes.rb

```diff
@@ -30,7 +30,6 @@
 
   get "*path(.:locale)(.:format)" => "content_items#show",
```

Comment:
I think if the only format allowed was atom, we can also remove the optional format from line 31?

Reply:
DONE! And fixed up.